apps/src/applab/applab.js

```diff
@@ -782,7 +776,8 @@ Applab.init = function (config) {
     showDebugButtons: showDebugButtons,
     showDebugConsole: showDebugConsole,
     showDebugSlider: showDebugConsole,
-    showDebugWatch: config.level.showDebugWatch || experiments.isEnabled('showWatchers')
+    showDebugWatch: config.level.showDebugWatch || experiments.isEnabled('showWatchers'),
+    debuggerUi,
```

Comment:
Sadly, it does have state. It holds a command history and an instance of the js interpreter.

At the moment it does not use redux for any of these, so the easiest way to get at the JsDebuggerUi instance without having to thread it through all the components, and without using other globals like the Applab or Gamelab singletons, is to put it into redux. pageConstants is not necessarily the right place for it though, and IMO, it should eventually be refactored to just store it's state in redux with it's own set of reducers and whatnot.

I could do that refactoring now as part of this PR. Or I could do it in a separate PR. Do you have a preference?